build.sbt

```diff
@@ -252,15 +289,22 @@ lazy val benchmarks = project
   .settings(publish / skip := true)
   .dependsOn(`play-jsonJVM`)
 
+val docsP = LocalProject("docs")
```

Comment:
`lazy` if cyclical?

Reply:
There's no need for this to be `lazy`.

If you mean make play-jsonJVM and docs lazy, then they already are.  And trying to use them directly gives you:

```
/d/play-json/build.sbt:308: error: recursive lazy value play-jsonJVM needs type
      val apiDocs = (doc in (`play-jsonJVM`, Compile)).value
          ^
sbt.compiler.EvalException: Type error in expression
[error] sbt.compiler.EvalException: Type error in expression
```

And giving them types gives you:

```
java.lang.StackOverflowError
	at sbt.internal.util.complete.Repeat.<init>(Parser.scala:938)
	at sbt.internal.util.complete.Parser$.checkRepeated$1(Parser.scala:297)
	at sbt.internal.util.complete.Parser$.repeat(Parser.scala:309)
	at sbt.internal.util.complete.Parser$.$anonfun$repeat$3(Parser.scala:305)
	at sbt.internal.util.complete.ValidParser.ifValid(Parser.scala:624)
	at sbt.internal.util.complete.ValidParser.ifValid$(Parser.scala:624)
	at sbt.internal.util.complete.ParserMain$$anon$2.ifValid(Parser.scala:383)
	at sbt.internal.util.complete.Parser$.repeat(Parser.scala:304)
	at sbt.internal.util.complete.Repeat.repeatDerive(Parser.scala:953)
	at sbt.internal.util.complete.Repeat.derive(Parser.scala:949)
	at sbt.internal.util.complete.MapParser.derive(Parser.scala:770)
	at sbt.internal.util.complete.ParserMain.loop$1(Parser.scala:485)
	at sbt.internal.util.complete.ParserMain.result(Parser.scala:487)
	at sbt.internal.util.complete.ParserMain.result$(Parser.scala:473)
	at sbt.internal.util.complete.Parser$.result(Parser.scala:138)
	at sbt.internal.util.complete.ParserMain.parse(Parser.scala:436)
	at sbt.internal.util.complete.ParserMain.parse$(Parser.scala:435)
	at sbt.internal.util.complete.DefaultParsers$.parse(Parsers.scala:345)
	at sbt.Project$.validProjectID(Project.scala:370)
	at sbt.Project$.sbt$Project$$unresolved(Project.scala:427)
	at sbt.Project$.apply(Project.scala:322)
	at $74f84aa39e44387c58ad$.docs$lzycompute(build.sbt:292)
	at $74f84aa39e44387c58ad$.docs(build.sbt:292)
	at $74f84aa39e44387c58ad$.play$minusjsonJVM$lzycompute(build.sbt:260)
	at $74f84aa39e44387c58ad$.play$minusjsonJVM(build.sbt:255)
	at $74f84aa39e44387c58ad$.docs$lzycompute(build.sbt:308)
	at $74f84aa39e44387c58ad$.docs(build.sbt:292)
	at $74f84aa39e44387c58ad$.play$minusjsonJVM$lzycompute(build.sbt:260)
```

So +/- this is the only way to do this.